library/core/src/alloc/layout.rs

```diff
@@ -514,4 +542,190 @@ mod verify {
             assert_eq!(layout.align(), a);
         }
     }
+
+    // pub const fn size(&self) -> usize
+    #[kani::proof]
```

Comment:
Why not `proof_for_contract`?

Reply:
Same problem as https://github.com/model-checking/verify-rust-std/pull/42#discussion_r1703939852